lib/optimization/table_data.go

```diff
@@ -81,10 +81,6 @@ func (t *TableData) RawName() string {
 	return t.name
 }
 
-func (t *TableData) Name(uppercaseEscNames bool, args *sql.NameArgs) string {
```

Comment:
Moved to https://github.com/artie-labs/transfer/pull/443/files#diff-25a5472065a253195427b0d5cca5e99df35797c1291b52cd6687d670a585b500R21